example.js

```diff
@@ -0,0 +1,7 @@
+var Micromark = require('.')
+
+var m = new Micromark()
+
+var res = m.write('# Hello!')
+
+console.log('res: ', res)
```

Comment:
It seems to me that `res` will always be "undefined"

Reply:
Yes, it is, this was just for debugging purposes 😉